extract.js

```diff
@@ -151,6 +151,17 @@ function loadBabelOpts(opts) {
 		}
 	}
 
+	// avoid conflicting with the legacy decorators plugin
+	if (opts.plugins && opts.plugins.findIndex((p) => p.key === 'proposal-decorators') > -1) {
```

Comment:
[question] It seems the code can be simplified using `some()` as below. What do you think?

```suggestion
	if (opts.plugins && opts.plugins.some((p) => p.key === 'proposal-decorators')) {
```

Reply:
It looks great!